tests/e2e/UsingGKE.md

```diff
@@ -81,8 +81,18 @@ export TAG="d0142e1afe41c18917018e2fa85ab37254f7e0ca"
 ```
 
 **Option 3:** Testing local changes
-
 If you want to test on uncommitted changes to master istio:
+
+***Option A*** Use in-cluster local registry
+If environment variable `HUB` is not set, local registry will be setup before test.
+All local codes will be compiled/dockered and pushed to it .
+
+* MacOS ONLY
+Add `docker.for.mac.localhost:5000` to insecure registries in the docker daemon setting before running the tests.
+Make sure you have followed this [doc](../local/localregistry/setup_bluemix.md) to configure docker daemons on all nodes.
```

Comment:
Why is this specific to bluemix?

Reply:
It is not required on GKE and minikube, since insecure registries is enabled for the service ip range.